apps/server/src/positions/positions.service.ts

```diff
@@ -34,6 +34,20 @@ export class PositionsService {
     return positions;
   }
 
+  /**
+   * Retrieve all positions with specified ids.
+   * @param ids list of position ids
+   * @returns all matching positions, hydrated
+   */
+  async findAllWithIds(ids: string[]) {
+    const positions = await this.prisma.position.findMany({
+      where: {
+        id: { in: ids },
+      },
+    });
```

Comment:
Suggestion?: should we throw an error if we can't find a position with the specified id